src/common/PythonManager.cpp

```diff
@@ -55,11 +55,23 @@ void PythonManager::initPythonHome()
     }
 #endif
 
+#if (PY_MAJOR_VERSION <= 3) && (PY_MICRO_VERSION < 11)
```

Comment:
Looking at the API and ABI versioning [documentation](https://docs.python.org/3/c-api/apiabiversion.html), I would suggest the following:
```suggestion
#if (PY_MAJOR_VERSION == 3) && (PY_MINOR_VERSION < 11)
```